pkg/mcp/snapshot/snapshot.go

```diff
@@ -107,7 +108,8 @@ func (si *StatusInfo) LastWatchRequestTime() time.Time {
 }
 
 // Watch returns a watch for an MCP request.
-func (c *Cache) Watch(request *source.Request, pushResponse source.PushResponseFunc, peerAddr string) source.CancelWatchFunc { // nolint: lll
+func (c *Cache) Watch(
```

Comment:
Might as well go all the way and wrap after each parameter and delete the `nolint: lll`. e.g.

```golang
func (c *Cache) Watch(
	request *source.Request, 
    pushResponse source.PushResponseFunc, 
    peerAddr string
) source.CancelWatchFunc { 
    group := g.groupIndex(...)
```